test/ventilation_test.h

```diff
@@ -6,3 +6,11 @@
 int startVentilation_afterOneInspiration_openInspiratoryValve();
 
 int startVentilation_duringInspiration_doesNotopenInspiratoryValve();
+
+int startVentilation_afterOneInspiration_setsVentilationPhaseToInspiration();
+
+int startVentilation_duringInspiration_doesNotChangeVentilationPhase();
+
+int startVentilation_afterTwoInspirations_updatesStartCurrentBreathCycle();
+
+int startVentilation_duringInspiration_doesNotUpdatesStartOfcurrentBreathCycle();
```

Comment:
There are some missing unit tests: 

1. There is no unit test probing behaviour for when `get_respiratory_rate() != STATUS_OK`, though this seems to be a pre-condition in `start_inspiration()`. 

2. There is no unit test to confirm that DSS is called if the pressure is greater than the maximum pressure.

Reply:
These are outside the scope not the same SRS SRS0010

Reply:
I think this misplaces the role of unit testing in my view. The unit tests should confirm the behaviour of the code as written, not necessarily tied one-to-one to requirements. Requirements based testing is essential, but it is different then unit testing.

Suggest adding associated unit tests for these cases.

Reply:
I have simplified the code